src/integrationTest/resources/internal_users.yml

```diff
@@ -12,3 +12,39 @@ admin:
   hash: "$2y$12$53iW.RRy.uumsmU7lrlp7OUCPdxz40Z5uIJo1WcCC2GNFwEWNiTD6"
   opendistro_security_roles:
   - "user_admin__all_access"
+rest-api-admin:
```

Comment:
In the integrationTest test package the model is focused around having test cases declare their setup instead of depending on magic values in config files, can we transition these resources over to that format?

Reply:
+1, it makes it easy to decipher the test when all data is located in a single place